eval.cpp

```diff
@@ -467,11 +489,22 @@ namespace Sass {
     }
     else
     {
-      rhs->is_delayed(false);
-      rhs = rhs->perform(this);
+      // rhs->set_delayed(false);
```

Comment:
Would like to keep it for some time, since it could be needed (and has no performance impacts).